src/app/rpmostree-builtin-status.cxx

```diff
@@ -1318,7 +1325,7 @@ print_history_entry (const rpmostreecxx::HistoryEntry &entry, GError **error)
         g_print ("  << Missing history information >>\n");
 
       /* XXX: factor out interesting bits from print_one_deployment() */
-      else if (!print_one_deployment (NULL, deployment, TRUE, FALSE, FALSE, NULL, NULL, NULL, NULL,
+      else if (!print_one_deployment (NULL, deployment, 0, FALSE, FALSE, NULL, NULL, NULL, NULL,
```

Comment:
Sounds reasonable, I added that. Since `index` now has a plethora of meanings depending on its value (don't print when < 0, "first" deployment when == 0, deployment index when >= 0), I added a short comment to the function. It looks a bit lost there because for one it only documents what `index` does (Because about most other things I can only speculate) and it seems to be pretty much the only docstring around... So should we leave it in there and maybe explain the other arguments as well? Or do you prefer to have the code consistent and remove the docs?